packages/packages.csv

```diff
@@ -173,6 +171,7 @@ Microsoft.Extensions.TaskCache.Sources,noship,exclude
 Microsoft.Extensions.TypeNameHelper.Sources,noship,exclude
 Microsoft.Extensions.WebEncoders,ship,include
 Microsoft.Extensions.WebEncoders.Sources,noship,exclude
+Microsoft.Extensions.WebSockets.Internal,noship,exclude
```

Comment:
It's a package with `.Internal`. We really shouldn't be publishing it to nuget.org